### ISSUE TITLE
File shows up twice in folder

### ISSUE THREAD

**Issue by liayn** [haiwen/seafile#1919]
![seafile duplicated entry](https://user-images.githubusercontent.com/222377/27256112-a2ea3ec8-53ac-11e7-96f8-7fae9ff06584.jpg)

Using the 6.0.6 windows client I recently got a situation where **a single file has been shown twice** in a folder. See screenshot. How the hell can that happen?

(I didn't check on the server directly unfortunately. I'm not sure if it is a client issue or a server issue, that's why I post it here.)

**Comment by freeplant** [comment]
Please check the web interface of the server side.

**Comment by liayn** [comment]
As I wrote above:

> I didn't check on the server directly unfortunately. I'm not sure if it is a client issue or a server issue, that's why I post it here.

Is it possible at all to create two files with the same name in the same folder on Windows? As far as I know the answer should be No. But I have no clue how seafile managed to do that.

**Comment by arjones85** [comment]
Windows will not let you do that. I guarantee it's a visual glitch of some sort, or the file name is different somehow that is imperceptible from this screenshot.

**Comment by liayn** [comment]
@arjones85 Okay, I'll declare this case closed, since I'm not able to reproduce.
For the record. We basically do not use the web interface, but solely rely on the client for Win/Mac/Android. This particular file was added to the library with my Windows client. I simply copied the file there, so I'm really not sure how any non-visual character should have made its way into the filename.

Maybe someone reopens this issue, if it also happens to someone else.
Thanks for the attention so far.
